dts/bindings/video/st,stm32-jpeg.yaml

```
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2025 STMicroelectronics.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+    STM32 JPEG Codec. The STM32 JPEG codec allows JPEG encoding and
+    decoding from / to MCU blocks which can be generated from NV12
+    format.
```

```suggestion
    STM32 JPEG Codec.
    
    The STM32 JPEG codec allows JPEG encoding and
    decoding from / to MCU blocks which can be generated from NV12
    format.
```

Maybe description can be made more generic too - from RM0486, it seems the JPEC codec is more general purpose than described here:
```suggestion
    STM32 JPEG Codec.
    
    The STM32 JPEG codec can decode JPEG-compressed images
    and encode uncompressed images to the JPEG format.
```
(Nit: `NV12` was confusing to me because RM only mentions `YUV`)